napalm/junos/junos.py

```diff
@@ -292,17 +293,92 @@ def compare_config(self):
 
     def commit_config(self, message="", revert_in=None):
         """Commit configuration."""
+        commit_args = {}
         if revert_in is not None:
-            raise NotImplementedError(
-                "Commit confirm has not been implemented on this platform."
-            )
-        commit_args = {"comment": message} if message else {}
+            if revert_in % 60 != 0:
+                raise CommitConfirmException(
```

Comment:
Before bailing out, I think we should also discard the loaded config and unlock the config database (otherwise we'll end up with a config DB being locked by a rogue session)

Reply:
Makes sense...that should be fixed.